rush.json

```diff
@@ -121,7 +121,7 @@
    * Specify a SemVer range to ensure developers use a Node.js version that is appropriate
    * for your repo.
    */
-  "nodeSupportedVersionRange": ">=12.13.0 <13.0.0",
+  "nodeSupportedVersionRange": ">=12.13.0 <15.0.0",

```

Comment:
validated on node v12.18.0
Did you test if v14 worked? my zsh export PATH is having issues locating rush on v14 `command not found: rush`

Reply:
Currently we can't, but in the future we can.